content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,232 @@
+$ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+{{< /text >}}
+
+The output from the above commands should include the DNS names of Galley and Sidecar Injector, respectively:
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+{{< /text >}}
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+{{< /text >}}
+
+## Enable webhook configurations
+
+1.  Generate `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` by running the following
+command. The YAML file [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml)
+contains an example DNS certificate configuration (details in [the certificate guide](/docs/tasks/security/dns-cert)).
+The following command uses the default Istio configuration plus the DNS certificate configuration in `values-istio-dns-cert.yaml`.
+The document of the helm template command can be found in the [link](https://helm.sh/docs/helm/#helm-template).
+
+    {{< text bash >}}
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
+        install/kubernetes/helm/istio > istio-webhook-config.yaml
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain MutatingWebhookConfiguration -->
+1.  From `istio-webhook-config.yaml`, search `'kind: MutatingWebhookConfiguration'` and save
+the `MutatingWebhookConfiguration` of Sidecar Injector to `sidecar-injector-webhook.yaml`. The following
+is a `MutatingWebhookConfiguration` in an example `istio-webhook-config.yaml`.
+
+    {{< text yaml >}}
+    apiVersion: admissionregistration.k8s.io/v1beta1
+    kind: MutatingWebhookConfiguration
+    metadata:
+      name: istio-sidecar-injector
+      labels:
+        app: sidecarInjectorWebhook
+        chart: sidecarInjectorWebhook
+        heritage: Tiller
+        release: istio
+    webhooks:
+      - name: sidecar-injector.istio.io
+        clientConfig:
+          service:
+            name: istio-sidecar-injector
+            namespace: istio-system
+            path: "/inject"
+          caBundle: ""
+        rules:
+          - operations: [ "CREATE" ]
+            apiGroups: [""]
+            apiVersions: ["v1"]
+            resources: ["pods"]
+        failurePolicy: Fail
+        namespaceSelector:
+          matchLabels:
+            istio-injection: enabled
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain ValidatingWebhookConfiguration -->
+1.  From `istio-webhook-config.yaml`, search `'kind: ValidatingWebhookConfiguration'` and save
```

Comment:
The actions the reader needs to take are not clear. How exactly can the reader determine what to copy and save in the new file? Where should that file be stored?

Reply:
Done.